spec/app/domain/services/update_course_exercise_exclusions/service_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'rails_helper'
+
+RSpec.describe Services::UpdateCourseExerciseExclusions::Service do
+
+  let(:service) { Services::UpdateCourseExerciseExclusions::Service.new }
+
+  let(:action_none) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_none_exclusions.exclusions
+  ) }
+
+  let(:action_some) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_some_exclusions.exclusions
+  ) }
+
+  let(:action_many) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_many_exclusions.exclusions
+  ) }
+
+  let(:action_repeat) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_some_exclusions.exclusions
+  ) }
+
+  let(:given_none_exclusions) {
+    build_stubbed(:exercise_exclusions, exclusions_count: number_of_exclusions_none,
```

Comment:
There are a bunch of things that rub me the wrong away about using `#build_stubbed` on a non-model.  We should discuss.